packages/interface/src/features/projects/components/ProjectItem.tsx

```diff
@@ -84,54 +77,58 @@ export const ProjectItem = ({
   const bannerImageUrl = recipient.bannerImageUrl ? recipient.bannerImageUrl : metadata.data?.bannerImageUrl;
   const profileImageUrl = recipient.profileImageUrl ? recipient.profileImageUrl : metadata.data?.profileImageUrl;
   const name = recipient.name ? recipient.name : metadata.data?.name;
-  let shortBio = recipient.shortBio ? recipient.shortBio : metadata.data?.shortBio;
   const bio = recipient.bio ? recipient.bio : metadata.data?.bio;
+  const impactCategory = recipient.impactCategory ? recipient.impactCategory : metadata.data?.impactCategory;
+
+  let shortBio = recipient.shortBio ? recipient.shortBio : metadata.data?.shortBio;
+
   if (!shortBio && bio) {
     shortBio = bio.substring(0, 140);
   }
-  const impactCategory = recipient.impactCategory ? recipient.impactCategory : metadata.data?.impactCategory;
+
+  if (isLoading) {
+    return (
+      <div className="flex h-[325px] w-full animate-pulse items-center justify-center rounded-[10px] bg-gray-100" />
+    );
+  }
 
   return (
     <Link href={`/rounds/${pollId}/${recipient.id}`}>
-      {isLoading && <Spinner className="h-16 w-16" />}
-
-      {!isLoading && (
-        <ProjectItemContent
-          actionButton={
-            state !== undefined &&
-            action &&
-            roundState === ERoundState.VOTING && (
-              <div className="flex justify-end pt-6">
-                <Skeleton>
-                  {state === EProjectState.DEFAULT && (
-                    <Button size="sm" variant="inverted" onClick={action}>
-                      Add to ballot
-                    </Button>
-                  )}
-
-                  {state === EProjectState.ADDED && (
-                    <Button size="sm" variant="primary" onClick={action}>
-                      Added
-                      <Image alt="check-white" height="18" src="/check-white.svg" width="18" />
-                    </Button>
-                  )}
-
-                  {state === EProjectState.SUBMITTED && (
-                    <Button size="sm" variant="disabled">
-                      Submitted
-                    </Button>
-                  )}
-                </Skeleton>
-              </div>
-            )
-          }
-          bannerImageUrl={bannerImageUrl}
-          impactCategory={impactCategory}
-          name={name}
-          profileImageUrl={profileImageUrl}
-          shortBio={shortBio}
-        />
-      )}
+      <ProjectItemContent
```

Comment:
<img width="439" alt="image" src="https://github.com/user-attachments/assets/41794c78-fa3a-4f60-9274-45571c04afe0" />

There is a width problem.